docs/source/usage/parameters.rst

```diff
@@ -527,6 +527,25 @@ This requires these additional parameters:
 * ``<element_name>.nslice`` (``integer``) number of slices used for the application of space charge (default: ``1``)
 
 
+``polygon_aperture``
+^^^^^^^^^^^^^^^^^^^^
+
+``polygon_aperture`` for a thin collimator element applying a transverse polygon aperture boundary defined by :math:`(x,y)` coordinates
+and optional radius below which all particles are transmitted. The vertices must define a closed curve and be ordered in the counter-clockwise direction.
```

Comment:
Closed curve: Below, you mention also that the last point provided by the user must be exactly the same as the first point. This is missing from the user-facing docs here, a user could assume that the last vertex automatically connects to the first.